### PR TITLE
Config for closing stale issues

### PR DESCRIPTION
**Pull request by max-sixty**
- [x] Closes https://github.com/pydata/xarray/issues/2681

Initially set at two years, and max rate of one / hour - we can make more aggressive when we know it's working

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7 # start with a large number and reduce over time
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+  - security
+  - "[Status] Maybe Later"
+
```
Comment (dcherian) [inline .github/stale.yml]: Add "feature_request" and "API_design" (not sure of syntax)? Those seem to hang around for a while.

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 700  # start with a large number and reduce shortly
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7 # start with a large number and reduce over time
```
Comment (fmaussion) [inline .github/stale.yml]: I would rather set to stale after one year, but set daysUntilClose to 4 weeks or so.

Comment (max-sixty) [reply .github/stale.yml]: I agree re one year, but let's start with longer and then reduce as we know it's working?

File: .github/stale.yml (hunk 1 above)
Comment (max-sixty) [reply .github/stale.yml]: I'll merge if you're OK with this @dcherian , unless there are any other comments?

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  - "[Status] Maybe Later"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
```
Comment (jhamman) [inline .github/stale.yml]: I think it would be reasonable for this to be true

File: .github/stale.yml (hunk 1 above)
Comment (dcherian) [reply .github/stale.yml]: Fine by me. Sorry for the delay in response.

**Comment by max-sixty** [comment]
This is now active, but rate limited at one per hour and with a two year tolerance. Let me know any thoughts and I'll adjust as necessary

**Pull request merged**